one-ui/src/main/java/com/marklogic/hub/oneui/auth/LoginLogoutHandler.java

```diff
@@ -19,13 +19,15 @@
 import org.springframework.security.web.WebAttributes;
 import org.springframework.security.web.authentication.AuthenticationSuccessHandler;
 import org.springframework.security.web.authentication.logout.LogoutSuccessHandler;
+import org.springframework.stereotype.Controller;
 
 import javax.servlet.ServletException;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 import javax.servlet.http.HttpSession;
 import java.io.IOException;
 
+@Controller
```

Comment:
Is there are more appropriate annotation to use here? Perhaps just "Component"? If that has the same effect, I think that'd be better because this class of course is not a controller.

Reply:
Actually, i think the handler should be a controller. It can be refactor into the com.marklogic.hub.oneui.controllers package to make it more clear.

Reply:
I agree with @rjrudin . I think it should be a "Component" and not a "Controller".

Reply:
OK, i do not have strong opinion with which one we should use for the spring annotation. I will check it because of majority takes over.